tiled/_tests/test_compression.py

```diff
@@ -0,0 +1,50 @@
+import numpy
+import pytest
+from starlette.testclient import TestClient
+
+from ..adapters.array import ArrayAdapter
+from ..adapters.mapping import MapAdapter
+from ..server.app import build_app
+
+
+@pytest.fixture
+def app():
+    metadata = {str(i): {str(j): j for j in range(100)} for i in range(100)}
+    tree = MapAdapter(
+        {
+            "compresses_well": ArrayAdapter.from_array(
+                numpy.zeros((1000, 1000)), metadata=metadata
+            )
+        },
+    )
+    return build_app(tree, authentication={"single_user_api_key": "secret"})
+    # In this module we use a raw TestClient instead of tiled.client to omit
+    # tiled.client's default headers and other configuration.
```

Comment:
I don't understand which headers/config are being bypassed, but I accept that you've thought about this.


I'd be interested in hearing about this later.